src/main/java/org/example/bookstore/model/Book.java

```diff
@@ -10,36 +10,21 @@
 import lombok.Data;
 
 @Entity
-@Table(name = "books")
 @Data
```

Comment:
Don't use `@Data` for entities, read which annotations it includes and use necessary